src/keys/bip44.rs

```diff
@@ -60,6 +60,30 @@ pub struct Bip44 {
 impl Bip44 {
     pub const PURPOSE: u32 = 44;
 
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    pub fn coin_type(mut self, s: u32) -> Self {
```

Comment:
Now that these aren't part of a builder, they should be named `with_x`.

Reply:
Renamed.